### PR TITLE
Navigation Bar - Animations

### PR DESCRIPTION
**Pull request by maov19**
Following features were implemented in Portfolio:

✔️ When the user clicks (or taps) the hamburger button, the mobile menu appears.
✔️ When the user clicks (or taps) the close (X) button, the mobile menu disappears.
✔️ When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
✔️ When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Comment by maov19** [comment]
Dear reviewer, 

Please consider that the collaboration made by my partner was previously merged from another branch (printscreen 🔽 )

Thanks, 



image

File: index.html [hunk 1]
```diff
@@ -19,21 +19,30 @@
...
         </ul>
     </nav>
-    </header>
+        <nav id="menuAll">
+        <ul>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+        </nav>
```
Comment (mckent05) [inline index.html]: - You currently have more than one `nav` tag in your portfolio, to help keep your project more semantically correct, I suggest that you have just one `nav` tag in your HTML page and style your nav using CSS media-queries to fit into either desktop or mobile screens. Please fix this 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @maov19, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (maov19) [reply index.html]: Fixed! 🔧 
The tag from the menu was changed.

**Comment by maov19** [comment]
Dear reviewer, 

The requested changes were implemented. ✏️  
Please let me now if there is something else I should work on.

**Review by binhussen — APPROVED** [review]
Hi @maov19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**